db/dbtest/benchmark.go

```diff
@@ -0,0 +1,142 @@
+package dbtest
+
+import (
+	"bytes"
+	"encoding/binary"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	dbm "github.com/cosmos/cosmos-sdk/db"
+)
+
+const strChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz" // 62 characters
+
+// RandStr constructs a random alphanumeric string of given length.
```

Comment:
I try to avoid using `goto` and `label` breaks whenever possible. Have you considered a similar and more general approach?

```go
var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")

func randSeq(n int) string {
    b := make([]rune, n)
    for i := range b {
        b[i] = letters[rand.Intn(len(letters))]
    }
    return string(b)
}
```

Reply:
This is carried over from tm-db without revisions, but yeah that would be much cleaner. However, now I realize this isn't used anywhere in the code that was imported, so I can just remove it.

Reply:
Even better! :-p